docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   ws:
-    image: golang
+    image: sprinkler
```

Comment:
would `build: .` instead of `image` work?

Reply:
that `build: . ` works.

Reply:
change committed